.chloggen/unreleased/dd-exporter-duplicate-tags.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: datadogexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Doesn't append duplicate ddtags on each log submission leading to 414 API errors."
+
+# One or more tracking issues related to the change
+issues: [16380]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext:
```

Comment:
Please make sure that no unrelated changes are included

Reply:
Not sure how that got in there! sorry